src/audit-license-headers.js

```diff
@@ -44,11 +46,27 @@ var COMMON_RAT_EXCLUDES = [
     '*.xcworkspacedata',
     '*.xccheckout',
     '*.xcscheme',
-    ];
+];
+
+var RAT_IGNORE_PATH          = '.ratignore';
+var RATIGNORE_COMMENT_PREFIX = '#';
+
+var RAT_NAME = 'apache-rat-0.10';
+var RAT_URL  = 'https://dist.apache.org/repos/dist/release/creadur/apache-rat-0.10/apache-rat-0.10-bin.tar.gz';
+
+function startsWith(string, prefix) {
+    return string.lastIndexOf(prefix, 0) === 0;
```

Comment:
Because `lastIndexOf` searches backwards, meaning that if it starts at `0`, it will check for one occurrence and return right away. Arguably, regular `indexOf` would be clearer though. Thoughts?